tools/npm/npm1.md

```diff
@@ -0,0 +1,158 @@
+# Definition
+
+### What is npm?
+**npm** is the **node package manager** that makes it easy for JavaScript developers to share the code that they've created to solve particular problems, and for other developers to reuse that code in their own applications.
```

Comment:
I like that we start with what is npm but the description is a bit generic - let's try to put something more solid that is going to be super clear for a newbie :slightly_smiling_face: